cocos/2d/CCActionManager.h

```diff
@@ -145,6 +145,24 @@ class CC_DLL ActionManager : public Ref
      */
     CC_DEPRECATED_ATTRIBUTE ssize_t numberOfRunningActionsInTarget(Node *target) const { return getNumberOfRunningActionsInTarget(target); }
 
+
+    /** Returns the numbers of actions that are running in a
+     *  certain target with a specific tag.
+     * Like getNumberOfRunningActionsInTarget Composable actions
+     * are counted as 1 action. Example:
+     * - If you are running 1 Sequence of 7 actions, it will return 1.
+     * - If you are running 7 Sequences of 2 actions, it will return 7.
+     *
+     * @param target    A certain target.
+     * @param tag       Tag that will be searched.
+     * @return  The numbers of actions that are running in a certain target
+     *          with a specific tag.
+     * @see getNumberOfRunningActionsInTarget
+     * @js NA
+     */
+    size_t getNumberOfRunningActionsInTargetByTag(const Node *target, int tag);
```

Comment:
@minggo I just used the same pattern of the "surrounding" methods.
Actually I didn't understand your comment "it is better to add const like ...", my PR 
is already using ```const Node *``, isn't it?

About the reference, I have no ideas what would be better, we have a lot of comments with this issue:
```
// FIXME: Passing "const O *" instead of "const O&" because HASH_FIND_IT requires the address of a pointer
// and, it is not possible to get the address of a reference
```

But this could go deeper in the engine architecture than I could make a clear statement about.
I leave to you guys decide what is better, regarding this modification.

Anyway is just let me know about that I'll happily make another PR with the changes!